wordpress-mobile/gutenberg-mobile
# Fix undo/redo not working on small changes on RichText

This PR fixes a problem where the undo/redo feature was not working if small changes were made to the content. Tapping `undo` does not update the text in the RichText powered block.

Actually the undo/redo feature was not working fine even if there were lot of changes to the RichText content, but in this case the problem was less noticeable, since the undo/redo feature replaced the content with the wrong text.

GB side PR: https://github.com/WordPress/gutenberg/pull/12417

To test this PR:
- Start the demo app
- Make a small change to a Para or Heading block
- Tap undo
- It should revert the content
- Start writing and deleting into RichText powered block, and have fun with the undo/redo feature. It should work.

- Please test also writing fast on Android.
